aQute.libg/src/aQute/libg/glob/Glob.java

```diff
@@ -141,4 +142,20 @@ public void getFiles(File root, List<File> result, boolean recursive, boolean us
 				getFiles(sub, result, recursive, usePath);
 		}
 	}
+
+	public static boolean in(Glob[] globs, String key) {
+		for (Glob g : globs) {
+			if (g.matcher(key).matches())
+				return true;
+		}
+		return false;
+	}
+
+	public static boolean in(Collection< ? extends Glob> globs, String key) {
+		for (Glob g : globs) {
```

Comment:
Isn't this more simply?

```java
return globs.stream().anyMatch(g -> g.matcher(key).matches());
```

Reply:
Maybe ... but I hate streaming when you debug. I do not think my code is less efficient (on the contrary I think) or harder to read. I loved streams but somehow I often revert back to old style to be able to single step it.

And I am not sure we should review these PRs for these coding styles because that can become a lot of work. I've no problem if you want to change these things since all works now but I really do not want to make these changes in this commit.

Reply:
You are right. This is a very complicated function... (sigh)

Never mind about my review.